packages/utils/src/instrument/fetch.ts

```diff
@@ -42,9 +55,12 @@ function instrumentFetch(): void {
         startTimestamp: timestampInSeconds() * 1000,
       };
 
-      triggerHandlers('fetch', {
-        ...handlerData,
-      });
+      // if there is no callback, fetch is instrumented
```

Comment:
```suggestion
      // if there is no callback, fetch is instrumented directly
```

Reply:
or something like this, the comment is a bit misleading because this always instruments fetch 😅 not sure what a good comment here would be xD